dev/doc/build-system.txt

```diff
@@ -89,7 +89,7 @@ enables partial recalculation of dependencies (only the dependencies
 of changed files are recomputed).
 
 If you add a dependency to a Coq camlp5 extension (grammar.cma or
-q_constr.cmo), then see sections ".ml4 files" and "new files".
+q_constr.cmo), then see sections ".mlg files" and "new files".
```

Comment:
This documentation file is in dire need of an update. Lots of comments are fairly outdated, including this one.

Reply:
I think this can be left for another PR, though